stake-pool/cli/src/main.rs

```diff
@@ -1626,18 +1630,14 @@ fn command_set_fee(
 
 fn command_list_all_pools(config: &Config) -> CommandResult {
     let all_pools = get_stake_pools(&config.rpc_client)?;
-    let count = all_pools.len();
-    for (address, stake_pool, validator_list) in all_pools {
-        println!(
-            "Address: {}\tManager: {}\tLamports: {}\tPool tokens: {}\tValidators: {}",
-            address,
-            stake_pool.manager,
-            stake_pool.total_lamports,
-            stake_pool.pool_token_supply,
-            validator_list.validators.len()
-        );
+    let cli_stake_pool_vec: Vec<CliStakePool> = all_pools.into_iter().map(|x| CliStakePool::from(x)).collect();
+    let cli_stake_pools = CliStakePools { pools: cli_stake_pool_vec };
+    match config.output_format {
+        OutputFormat::Display => println!("{}", config.output_format.formatted_string(&cli_stake_pools)),
+        OutputFormat::Json => println!("{}", config.output_format.formatted_string(&cli_stake_pools)),
+        OutputFormat::JsonCompact => println!("{}", config.output_format.formatted_string(&cli_stake_pools)),
+        _ => {}
```

Comment:
Any chance we could also do this on the normal `command_list_pool`?

Reply:
Yep good point. I will do it.

Reply:
Is it ok for you if I do it in a second PR?

Reply:
Almost done.